guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -3,10 +3,7 @@
 
 Use this procedure for a {ProjectServer} with access to the public internet
 
-[WARNING]
-If you customize configuration files, manually or using a tool such as Hiera, these changes are overwritten when the installation script runs during upgrading or updating.
-You can use the `--noop` option with the {foreman-installer} script to test for changes.
-For more information, see the Red Hat Knowledgebase solution https://access.redhat.com/solutions/3351311[How to use the noop option to check for changes in {Project} config files during an upgrade.]
+include::common/modules/snip_using_installer_noop.adoc[]
```

Comment:
```suggestion
include::../common/modules/snip_using_installer_noop.adoc[]
```
Looks like this is failing the build. Alternatively, a symlink to common could be added in topics. The long-term solution would be of course to move all modules to common but that would require separate PR.